src/main/java/com/kodilla/ecommercee/User.java

```diff
@@ -0,0 +1,47 @@
+package com.kodilla.ecommercee;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue
+    @Column(unique = true)
+    private Long id;
+
+    @Column
+    private String firstname;
+
+    @Column
+    private String lastname;
+
+    @Column
+    private String address;
+
+    @Column
+    private Integer postcode;
+
+    @OneToOne (fetch = FetchType.EAGER)
```

Comment:
Sprawdź domyślny fetchType przy mapowaniu OneToOne. Nie jest to przypadkiem właśnie EAGER?

Reply:
Jest to EAGER :-) ale jakiś czas temu było inaczej. Więc ja tam akurat czasem wolę napisać...

Reply:
:)